store/modules/events.ts

```diff
@@ -62,8 +63,8 @@ export default {
 
       state[events] = eventsList
     },
-    setActiveTab (state, payload) {
-      state.activeTab = payload
+    setCommunityEventsShowed (state, payload) {
```

Comment:
You can use your new type to annotate `payload` and ensure it is called with one of the possible values.